### ISSUE TITLE
CodeMaid adding tabs that shouldn't be there

### ISSUE THREAD

**Issue by jws305** [codecadwallader/codemaid#858]
### Environment
- Visual Studio version: 2022 Professional
- CodeMaid version: 12.0
- Code language: C#

### Description
Cleanup Active Document causes block to add an indent every time it is run, i.e. if I run the task 5 times, I get 5 tabs in front of the block.

### Steps to recreate
1. Clone https://github.com/jws305/CodeMaidIndentIssue
2. Run Cleanup Active Document on Program.cs

### Current behavior
Erroneous tabs added

### Expected behavior
The result of the cleanup should be deterministic.

**Comment by AndreiYakovlev** [comment]
Me too but codemaid adding spaces

**Comment by codecadwallader** [comment]
Thanks for reporting the issue.  I am able to reproduce it and that is unexpected.  I suspect it is something specific with this code segment:

```
        private static Lazy<int> Item { get; } = new Lazy<int>(() =>
        {
            return 1;
        });
```

**Comment by codecadwallader** [comment]
It looks to be tied to the insert explicit access modifiers logic on properties.  If that option is turned off the behavior stops.

**Comment by DamianSuess** [comment]
It goes without saying, this is also affecting VS 2019 (16.11.4).

**Comment by DamianSuess** [comment]
To validate, this is the symptom of the bug, correct?  The example is from our ViewModel using Prism Library, which as you can imagine uses a lot of `DelegateCommand` Properties.

**Before:**
```cs
    public DelegateCommand OnShowMain => new DelegateCommand(async () =>
    {
        await NavigationService.NavigateAsync($"../{nameof(MainView)}");
    });
```

**After:** (_note single space indention after Ctrl+M+Space_)
```cs
    public DelegateCommand OnShowMain => new DelegateCommand(async () =>
     {
         await NavigationService.NavigateAsync($"../{nameof(MainView)}");
     });
```

**Comment by codecadwallader** [comment]
Yes, I believe if you disable the options around inserting explicit access modifiers that will bypass the SDK bug and avoid the issue.

**Comment by wx0322** [comment]
> Yes, I believe if you disable the options around inserting explicit access modifiers that will bypass the SDK bug and avoid the issue.

thanks, solve my problem.
